otellog/otel_log.go

```diff
@@ -0,0 +1,70 @@
+package otellog
+
+import (
+	"fmt"
+	"io"
+	"sync"
+	"time"
+)
+
+type Logger interface {
+	Log(level LogLevel, msg fmt.Stringer)
```

Comment:
```suggestion
	Log(ctx context.Context, level LogLevel, msg fmt.Stringer, attrs ...attribute.KeyValue)
```

What about this interface?  Including the `Context` enables the logger to correlate entries with active spans or other context data.  Including attributes allows for something more like structured logging.

Reply:
Yes. I think we can add context and attribute to the interface method.

Reply:
@Aneurysm9 , @bhautikpip : In structured logging I used to see an hierarchy of values. Like: in some function we added a value and called another function, then in that function we added a value more and called a logger.

And I do not understand how it supposed to work in this interface. Is the hierarchy of values stored in the context?